src/settings/Settings.ts

```diff
@@ -424,6 +424,10 @@ export const SETTINGS: {[setting: string]: ISetting} = {
         supportedLevels: LEVELS_DEVICE_ONLY_SETTINGS_WITH_CONFIG,
         default: "en",
     },
+    "spell-check-languages": {
+        supportedLevels: LEVELS_DEVICE_ONLY_SETTINGS_WITH_CONFIG,
+        default: ["en-US"],
```

Comment:
If there is a setting, I suppose it should default to en-GB, to match Element's UK English default...?